### PR TITLE
Fix MainThreadCheckerRuntime.cpp to recognize properties from functions.

### PR DESCRIPTION
**Pull request by kubamracek**
Fix MainThreadCheckerRuntime.cpp to recognize properties from functions.

<rdar://problem/36557167>

### CODE REVIEW

**Comment by kubamracek** [comment]
@swift-ci please test

File: source/Plugins/InstrumentationRuntime/MainThreadChecker/MainThreadCheckerRuntime.cpp [hunk 1]
```diff
@@ -112,6 +112,7 @@ static std::string TranslateObjCNameToSwiftName(std::string className,
       swift::ClassDecl *cls = llvm::dyn_cast<swift::ClassDecl>(VD);
       if (!cls)
         return;
+      cls->loadAllMembers();
```
Comment (jrose-apple) [inline source/Plugins/InstrumentationRuntime/MainThreadChecker/MainThreadCheckerRuntime.cpp]: This should not be necessary; `lookupDirect` should handle it. If that's not the case we should change something in Swift, not in MainThreadChecker.

Comment (kubamracek) [reply source/Plugins/InstrumentationRuntime/MainThreadChecker/MainThreadCheckerRuntime.cpp]: Yes, it is not the case. The lookup started returning wrong results after <https://github.com/apple/swift/pull/13866> landed. Do you have a sense of what's going on?

Comment (jrose-apple) [reply source/Plugins/InstrumentationRuntime/MainThreadChecker/MainThreadCheckerRuntime.cpp]: Not obviously, but maybe there's a problem deserializing an AccessorDecl without its accompanying property? (See ModuleFile::loadObjCMethods.)

**Comment by kubamracek** [comment]
Please test with following pull request:
https://github.com/apple/swift/pull/17150

@swift-ci Please test

**Comment by kubamracek** [comment]
Please test with following pull request:
apple/swift#17150

@swift-ci Please test

**Comment by kubamracek** [comment]
@jrose-apple so it this okay now?

**Comment by jrose-apple** [comment]
Yep! I mean, now you're just throwing out the unnecessary code. :-)

**Comment by kubamracek** [comment]
Please test with following pull request:
apple/swift#17193

@swift-ci Please test

**Pull request merged**